integration-test/src/test/java/org/sagebionetworks/IT520SynapseJavaClientEvaluationTest.java

```diff
@@ -312,7 +319,53 @@ public void testEvaluationRoundTrip() throws SynapseException, UnsupportedEncodi
 		
 		assertEquals(initialCount, synapseOne.getAvailableEvaluationsPaginated(100, 0).getResults().size());
 	}
-	
+
+	@Test
+	public void testEvaluationRound_RoundTrip() throws SynapseException {
+		eval1 = synapseOne.createEvaluation(eval1);
+		evaluationsToDelete.add(eval1.getId());
+
+		Instant now = Instant.now();
+		EvaluationRound round = new EvaluationRound();
+		round.setEvaluationId(eval1.getId());
+		round.setRoundStart(Date.from(now));
+		round.setRoundEnd(Date.from(now.plus(1, ChronoUnit.DAYS)));
+
+		//create
+		EvaluationRound created = synapseOne.createEvaluationRound(round);
+
+		//read
+		EvaluationRound retrieved = synapseOne.getEvaluationRound(created.getEvaluationId(), created.getId());
+		assertEquals(created, retrieved);
+
+		//read all
+		EvaluationRound round2 = new EvaluationRound();
+		round2.setEvaluationId(eval1.getId());
+		round2.setRoundStart(Date.from(now.plus(1, ChronoUnit.DAYS)));
+		round2.setRoundEnd(Date.from(now.plus(2, ChronoUnit.DAYS)));
+		EvaluationRound created2 = synapseOne.createEvaluationRound(round2);
+
+		EvaluationRoundListResponse listResponse = synapseOne.getAllEvaluationRounds(created.getEvaluationId(), new EvaluationRoundListRequest());
+		assertEquals(Arrays.asList(created, created2), listResponse.getPage());
+
```

Comment:
I would probably add also a simple test with a submission, checking that the correct round was set. I would update the limit and make sure that you cannot submit anymore.